src/main/java/appeng/integration/modules/Waila.java

```diff
@@ -51,15 +64,11 @@ public static void register( IWailaRegistrar registrar )
 	@Override
 	public void init() throws Throwable
 	{
-		this.testClassExistence( IWailaDataProvider.class );
-		this.testClassExistence( IWailaRegistrar.class );
-
 		FMLInterModComms.sendMessage( "Waila", "register", this.getClass().getName() + ".register" );
 	}
 
 	@Override
 	public void postInit()
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1186)